src/charm.py

```diff
@@ -143,8 +143,8 @@ def _setup_db_schemas(self, event):
             return
 
         schema_dirs = {
-            "db": "/etc/temporal/schema/postgresql/v96/temporal/versioned",
-            "visibility": "/etc/temporal/schema/postgresql/v96/visibility/versioned",
+            "db": "/etc/temporal/schema/postgresql/v12/temporal/versioned",
```

Comment:
Is the `v12` newer than `v96`, how is this an update?

Reply:
`v12` is the newer schema version, created as part of the v1.20.0 release to allow PostgreSQL 12+ DBs to be used for advanced visibility. See [here](https://github.com/temporalio/temporal/commit/cb22aab7c4204743a8e5b419dafaeb1137d705e3).